MapboxVisionAR/Render/Renderer/Error/ARRendererError.swift

```diff
@@ -0,0 +1,11 @@
+/// The error that occurred during rendering process.
+enum ARRendererError: LocalizedError {
+    /// A queue that organizes command buffers to be executed by a GPU can't be created.
```

Comment:
Maybe we should provide the text of the comment via `errorDescription` property of `LocalizedError` protocol

Reply:
Updated.